codewell-fe/src/Components/Comment/CommentCard.js

```diff
@@ -18,14 +19,19 @@ const commentStyle = {
  * - used to display a comment
  * Receives
  *   - comment data with the following props
+ *     - id - the id of the comments list which includes the parent comments and
+ *        their replies
  *     - userId
  *     - datePosted
  *     - body
  *     - codeId
  *     - commentIndex
  *     - parentCommentId
- *   - updateComments - function that updates the comments
  *   - isCodeLineComment - 1 if is a code line comment, 0 otherwise
+ *   - updateComments - function that updates the comments
+ *   - isParentComment - it's defined if it's a parent comment. Otherwise,
+        it's undefined
```

Comment:
```suggestion
         it's undefined
```
also you forget the '*' and the beginning of the comment line